cfme/tests/cloud/test_providers.py

```diff
@@ -502,7 +503,10 @@ def test_azure_subscription_required(request, provider):
 @pytest.mark.tier(2)
 @test_requirements.azure
 @pytest.mark.provider([AzureProvider], scope="function", override=True, selector=ONE)
```

Comment:
I guess you don't need override here.

Reply:
@izapolsk I got 
```
cfme/markers/env_markers/provider.py:428: in process_env_mark
    "specifying the override flag [{}]".format(metafunc.function.__name__)
E   Exception: You have an override provider without specifying the override flag [test_azure_multiple_subscription]
```

Reply:
@lina-nikiforova, weird, sorry, I was told that override could be removed as redundant after recent changes.
@mshriver, is this an issue in provider marker or I'm just wrong ?